ArcGISRuntimeSDKQt_CppSamples/Maps/ShowLocationHistory/ShowLocationHistory.cpp

```diff
@@ -0,0 +1,131 @@
+}
+
+ShowLocationHistory::ShowLocationHistory(QObject* parent /* = nullptr */):
+  QObject(parent),
+  m_map(new Map(Basemap::darkGrayCanvasVector(this), this)),
```

Comment:
If the sample is going to use an off road trail, Imagery or Terrain basemaps may be better. The dark gray could make it seem like it is randomly moving around the map. Alternatively, you could keep this basemap and come up with a different polyline to track that is on a road